pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -637,6 +637,19 @@ func (p *packetParser) processRecord(ctx context.Context, id int) {
 				p.enricher.Write(ev)
 			}
 
+			// Add conntrack metrics.
```

Comment:
this doesn't look right to me. If there are multiple connections of the same type and direction, when packetparser receives each packet event from each connection, would this just "reset" the packet and byte metrics for this node to whatever the counters of the connection the packet belongs to? Let's say that I have connection A and B, and packetparser is receiving packets from A for every other packet from B, the node metrics would flip flop

Reply:
Thanks @nddq ! I'll look into implementing this inside conntrack GC loop to increment connection-level counters based on BPF conntrack metrics. Then we can set the metrics values as node-level (basic) and pod-level (advanced) metrics. I'll push another commit soon.